handlers/main.yml

```diff
@@ -5,3 +5,13 @@
     name: '{{ item }}'
     state: restarted
   with_items: '{{ iptables_services }}'
+
+- name: rsyslog restart
+  service:
+    name: 'rsyslog'
+    state: restarted
+
+- name: logrotate restart
```

Comment:
At least on Debian, logrotate is triggered through a systemd timer, there is no permanently running service to restart.